doc/manual/src/contributing/experimental-flags.md

```diff
@@ -0,0 +1,89 @@
+This section aims at describing the notion of a Nix "experimental
+flag", and how it fits into the big picture of the developpement of Nix.
+
+# What are these xp flags?
+
+Since Nix 2.4, some features of Nix can be flagged as
+experimental.Conceptually, this means that these features are not
+considered as part of Nix proper, but are only included in the code to
+make their developpement and testing easier.Practically, these require
+an explicit opt-in, by adding the name of the experimental feature to
+the `experimental-features` configuration option. This is really handy
+as it makes it harder to accidentally use an experimental feature
+without knowing it.
```

Comment:
```suggestion
Since Nix 2.4, some features of Nix can be flagged as
experimental. This means that they can be changed or removed at any time.
Users must explicitly opt in to them by adding the name of the experimental feature to
the `experimental-features` configuration option. This is really handy
as it makes it harder to accidentally use an experimental feature
without knowing it.
```